nautobot/core/views/mixins.py

```diff
@@ -71,6 +75,7 @@ class NautobotViewSetMixin(GenericViewSet, AccessMixin, GetReturnURLMixin, FormV
     # serializer_class has to be specified to eliminate the need to override retrieve() in the RetrieveModelMixin for now.
     serializer_class = None
     table_class = None
+    notes_form_class = NoteForm
```

Comment:
Why is this defined here? I don't see it being used anywhere else?

Reply:
get_form_class uses it. Glenn asked me to move it here.

Reply:
Ah okay, I didn't catch that before. Thanks!

```python
            form_class = getattr(self, f"{self.action}_form_class", None)
```